programs/survival/README.md

```diff
@@ -133,6 +144,11 @@ Various scripts that modify various game elements, often replicating popular mod
 	Left clicking will send your items to any unfilled slots in the inventory.
 	(Please add a better descreption as right now it just sounds like a less laggy storage_drawers.sc)
 
+### [item_holding_mobs_despawn.sc](https://github.com/gnembon/scarpet/blob/master/programs/survival/item_holding_mobs_despawn.sc):
+#### By altrisi
```

Comment:
I'm not the author, just changed the file name and therefore I'm mentioned the last.

Reply:
Ah. Mah bad